components/ModEditHealthcareProfessionalSection.vue

```diff
@@ -0,0 +1,279 @@
+<template>
+    <div class="mod-healthcare-professional-section">
+        <h2
+            class="mb-3.5 text-start text-primary-text text-2xl font-bold font-sans leading-normal"
+        >
+            {{ $t('modHealthcareProfessionalSection.healthcareProfessionalNameHeading') }}
+        </h2>
+        <div class="flex flex-col my-4">
+            <ModInputField
+                v-model="healthcareProfessionalsStore.healthcareProfessionalSectionFields.lastName"
+                data-testid="mod-healthcare-professional-section-lastName"
+                :label="$t('modHealthcareProfessionalSection.labelHealthcareProfessionalLastName')"
+                type="text"
+                :placeholder="$t('modHealthcareProfessionalSection.placeholderTextHealthcareProfessionalLastName')"
+                :required="true"
+            />
+            <ModInputField
+                v-model="healthcareProfessionalsStore.healthcareProfessionalSectionFields.localizedFirstName"
+                data-testid="mod-healthcare-professional-section-firstName"
+                :label="$t('modHealthcareProfessionalSection.labelHealthcareProfessionalFirstName')"
+                type="text"
+                :placeholder="$t('modHealthcareProfessionalSection.placeholderTextHealthcareProfessionalFirstName')"
+                :required="true"
+            />
+            <ModInputField
+                v-model="healthcareProfessionalsStore.healthcareProfessionalSectionFields.localizedMiddleName"
+                data-testid="mod-healthcare-professional-section-middleName"
+                :label="$t('modHealthcareProfessionalSection.labelHealthcareProfessionalMiddleName')"
+                type="text"
+                :placeholder="$t('modHealthcareProfessionalSection.placeholderTextHealthcareProfessionalMiddleName')"
+                :required="false"
+            />
+            <label
+                for="Name Locales"
+                class="my-2 text-primary-text text-sm font-bold font-sans"
+            >
+                {{ $t('modHealthcareProfessionalSection.labelHealthcareProfessionalNameLocale') }}
+            </label>
+            <select
+                v-model="healthcareProfessionalsStore.healthcareProfessionalSectionFields.locale"
+                data-testid="mod-healthcare-professional-section-name-locale"
+                name="Name Locales"
+                class="mb-5 px-3 py-3.5 w-96 h-12 bg-secondary-bg rounded-lg border border-primary-text-muted
+                            text-primary-text text-sm font-normal font-sans placeholder-primary-text-muted"
+            >
+                <option
+                    v-for="(locale, index) in Locale"
+                    :key="`${locale}-${index}`"
+                >
+                    {{ locale }}
+                </option>
+            </select>
+            <button
+                v-show="moderationScreenStore.activeScreen === ModerationScreen.EditSubmission"
```

Comment:
I could be wrong about this, but I think @ermish on a convention that conditionals such as `v-show` should be passed a variable.